modules/sqs-refined/src/main/scala/com/rewardsnetwork/pureaws/sqs/refined/RefinedSqsClient.scala

```diff
@@ -47,10 +48,19 @@ trait RefinedSqsClient[F[_]] {
     */
   def sendMessage(queueUrl: String, messageBody: String, delaySeconds: Int Refined DelaySeconds): F[String]
 
+  /** Send a message with attributes to an SQS queue. Message delay is determined by the queue settings
+    * @return
+    *   The message ID string of the sent message.
+    */
+  def sendMessage(
+      queueUrl: String,
+      messageBody: String,
+      messageAttributes: Map[String, MessageAttributeValue]
+  ): F[String]
```

Comment:
It might be better to add a parameter `messageAttributes: Map[String, MessageAttributeValue] = Map.empty` to the other `sendMessage` methods, to keep the ability to add delay in seconds manually.

Reply:
Same for all of the other instances of this as well, below.